### PR TITLE
Add module history information

### PR DESCRIPTION
**Pull request by kevinstratford**
For openfoam/com/v2106

### CODE REVIEW

File: docs/research-software/openfoam/openfoam.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ need to be added to any job/batch submission scripts you want to use to
 run OpenFOAM. Note that all the centrally installed versions of OpenFOAM
 are compiled under `PrgEnv-gnu`.
 
+Note there are no default versions specificied. It is recommended to
```
Comment (jsindt) [inline docs/research-software/openfoam/openfoam.md:59]: I'm not sure what you mean here -- if not specified, the default version is the last one to be compiled (in this case the .org v8 version).

**Review by jsindt — COMMENTED** [review]
I have a minor question about the mention of default versions. Otherwise, looks good to me.

File: docs/research-software/openfoam/openfoam.md (hunk 1 above)
Comment (kevinstratford) [reply docs/research-software/openfoam/openfoam.md]: "module versions" may be more clear.

**Review by jsindt — APPROVED** [review]
LGTM

**Pull request merged**